src/web_fetch.js

```diff
@@ -135,40 +150,70 @@ const readContentFromFile = async (openedFile, defaultMimeType, opts) => {
   }
 
   if (multipart) {
-    mimeType = 'multipart/byteranges';
+    mimeType = MIME_TYPE_BYTERANGES;
   }
 
   const response = {
     headers: {
-      'Content-Type': mimeType
+      [HEADERS_CONTENT_TYPE]: mimeType
     },
     body: data
   };
 
   if (range && multipart) {
-    response.headers['Content-Length'] = JSON.stringify(data).length;
+    response.headers[HEADERS_CONTENT_LENGTH] = JSON.stringify(data).length;
     delete response.body;
     response.parts = data;
   } else if (range) {
     endByte = (end === fileSize - 1) ? fileSize - 1 : end;
-    response.headers['Content-Length'] = lengthToRead;
-    response.headers['Content-Range'] = `bytes ${start}-${endByte}/${fileSize}`;
+    response.headers[HEADERS_CONTENT_LENGTH] = lengthToRead;
+    response.headers[HEADERS_CONTENT_RANGE] = `bytes ${start}-${endByte}/${fileSize}`;
   }
   return response;
 };
 
+async function fetchHelper(url) {
```

Comment:
@bochaco can you document this func a little as it's unclear from the naming what it's doing exactly.

Reply:
sure @joshuef , I'll push a commit with a comment for it